source/declarations.tex

```diff
@@ -8363,7 +8363,7 @@
 \tcode{extern}
 specifier\iref{dcl.stc} for the purpose of determining the linkage of the
 declared name and whether it is a definition. Such a declaration shall
-not specify a storage class.
+not contain a \grammarterm{storage-class-specifier}.
```

Comment:
"contain" is too general. This could be read as disallowing `extern "C" auto x = []() mutable {};` because it contains the *storage-class-specifier* `mutable`. Maybe "have" instead of "contain"?